src/minErr.js

```diff
@@ -82,6 +82,11 @@ function isValidObjectMaxDepth(maxDepth) {
 
 function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
+
+  var url = 'https://errors.angularjs.org/"NG_VERSION_FULL"/';
+  var regex = url + '[\\s\\S]*';
```

Comment:
Minor, but the RegExp will match any character in place of the `.`s that appear in the URL.

Reply:
Good catch - I'll change it